src/main/steps/respondent/yourdocuments/alldocuments/medicalreports/template.njk

```diff
@@ -21,7 +21,7 @@
               {{doc.createdDate}}
             </dt>
             <dd class="govuk-summary-list__value">
-              <a href={{doc.href}}>{{ doc.fileName }}</a>
```

Comment:
why are there lot of templates repeating  the same code and duplicated? use one template instead

Reply:
Yes vivek created one common template for all docs of respondent , applicant. Placed it in common folder as ... common/alldocuments/template.njk